CMakeLists.txt

```diff
@@ -96,7 +96,8 @@ endif()
 set(oxenmq_build_static_libzmq OFF)
 if(TARGET libzmq)
     target_link_libraries(oxenmq PUBLIC libzmq)
-elseif(BUILD_SHARED_LIBS)
```

Comment:
the reason it is checking `if(TARGET libzmq)` before this is to see if a parent cmake project already has a libzmq target set up and to use it if it does.
so this `elseif(BUILD_SHARED_LIBS)` is correct logic here.

Reply:
`pkg_check_modules(libzmq libzmq>=4.3 IMPORTED_TARGET)` is probably the part that is somehow borked sometimes like if `pkg-config` is not available.

Reply:
Ah right, I found it a bit confusing first, you're saying OFF should be with bundled and ON just the system libs ?

In that case, I got some issues on my system where it can't find either my own libzmq version, let me see